app/src/main/res/layout/fragment_full_announcement.xml

```diff
@@ -91,12 +91,20 @@
                 android:paddingTop="10dp"
                 >
 
-                <TextView
-                    android:id="@+id/announcement_content"
+                <ScrollView
                     android:layout_width="match_parent"
-                    android:layout_height="match_parent"
-                    android:text="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque mollis, lectus id dictum blandit, elit elit auctor metus, in ultrices velit justo in odio. Maecenas metus nibh, elementum quis orci sit amet, sodales tincidunt est. Nam mollis aliquet lorem ac sodales. Pellentesque posuere convallis tellus, vitae dignissim felis fermentum sit.Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque mollis, lectus id dictum blandit, elit elit auctor metus, in ultrices velit justo in odio. Maecenas metus nibh, elementum quis orci sit amet, sodales tincidunt est. Nam mollis aliquet lorem ac sodales. Pellentesque posuere convallis tellus, vitae dignissim felis fermentum sit."
-                    android:textSize="15sp" />
+                    android:layout_height="match_parent">
+
+                    <TextView
+                        android:id="@+id/announcement_content"
+                        android:layout_width="match_parent"
+                        android:layout_height="match_parent"
+                        android:text="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque mollis, lectus id dictum blandit, elit elit auctor metus, in ultrices velit justo in odio. Maecenas metus nibh, elementum quis orci sit amet, sodales tincidunt est. Nam mollis aliquet lorem ac sodales. Pellentesque posuere convallis tellus, vitae dignissim felis fermentum sit.Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque mollis, lectus id dictum blandit, elit elit auctor metus, in ultrices velit justo in odio. Maecenas metus nibh, elementum quis orci sit amet, sodales tincidunt est. Nam mollis aliquet lorem ac sodales. Pellentesque posuere convallis tellus, vitae dignissim felis fermentum sit."
```

Comment:
If you're using the `text` attribute to preview what the view looks like in Android Studio, then always use `tools:text` instead. `android:text` actually sets the `TextView` text at runtime, which is a waste of resources.